midend/Makefile.am

```diff
@@ -23,6 +23,7 @@ midend_UNIFIED = \
 	midend/nestedStructs.cpp \
 	midend/inlining.cpp \
 	midend/interpreter.cpp \
+	midend/isolateMethodCalls.cpp \
```

Comment:
This pass cannot be part of the mid-end, it has to be in the back-end if anywhere at all.